### PR TITLE
Purescript

### PR DESCRIPTION
**Pull request by rizary**
This code finally build on pulp 0.11. It still lacks of api implementation on tags and I am still working on it.

### CODE REVIEW

File: purescript/src/#Main.purs# [hunk 1]
```diff
@@ -0,0 +1,970 @@
+module Main (main) where
```
Comment (bergmark) [inline purescript/src/#Main.purs#]: Accidentally added, you can add `#*` to .gitignore

**Comment by hvr** [comment]
@bergmark btw, I think we should also integrate building the purescript code via Travis CI; is there some examples which I can steal from?

**Comment by bergmark** [comment]
I haven't looked into that.

File: purescript/src/#Main.purs# (hunk 1 above)
Comment (hvr) [reply purescript/src/#Main.purs#]: @bergmark btw, I think you need to escape `#`s

**Comment by rizary** [comment]
@hvr and @bergmark 

This pull is for /packages page. I think for this page, it is already works as expected. I notice in the production site, when you sort Tags and then try to sort with prefix, the list of tagged package was not sorted by the prefix. I don't know whether it supposed to work like that at the first place. But this commit fixed that.

**Pull request merged**